Dockerfile

```diff
@@ -13,6 +13,14 @@ RUN pip install -r requirements-dev.txt
 WORKDIR /build/
 ADD . /build/
 
+# The following command replaces the @VERSION@ string in setup.py and package.json
+# with the tagged version number from GIT_TAG
+ENV VERSION="0.0.0"
```

Comment:
Is 0.0.0 better than the existing 0.0.1?

Reply:
I wanted to be explicitly clear that this was not an actual version.